README.rst

```diff
@@ -40,8 +40,8 @@ A Simple Example
     db = SQLAlchemy(app, model_class=Base)
 
     class User(db.Model):
-        id: Mapped[int] = mapped_column(db.Integer, primary_key=True)
```

Comment:
Can mapped_column import be removed now?

Reply:
No,it's still used for set `id` field to primary key.